scripts/packages/VSCodeServer/src/display.jl

```diff
@@ -1,105 +1,50 @@
 struct InlineDisplay <: AbstractDisplay end
```

Comment:
nit, it feels like `InlineDisplay` is now better to be renamed to `VSCodeDisplay` or such.

Reply:
Yeah, true. Not too bad though because it's not exported and `VSCodeServer.InlineDisplay` makes some sense ;)